src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/statistikk/saksstatistikk/SakStatistikkServiceTest.kt

```diff
@@ -44,7 +44,7 @@ class SakStatistikkServiceTest : OppslagSpringRunnerTest() {
         stegService.utførSteg(behandling.id, BehandlingSteg.REGISTRERE_PERSONGRUNNLAG)
         assertEquals(
             1,
-            taskService.taskRepository.findAll().filter { it.type == SendBehandlinghendelseTilDvhTask.TASK_TYPE }.count()
+            taskService.findAll().filter { it.type == SendBehandlinghendelseTilDvhTask.TASK_TYPE }.count()
```

Comment:
Gammel kode, men dette kan skrives om til 
```suggestion
            taskService.findAll().count { it.type == SendBehandlinghendelseTilDvhTask.TASK_TYPE }
```